jtec-instrumentation/src/main/java/edu/tum/sse/jtec/instrumentation/systemevent/SysEventWriter.java

```diff
@@ -0,0 +1,27 @@
+package edu.tum.sse.jtec.instrumentation.systemevent;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+
+public class SysEventWriter {
+
+    public static void writeMessage(final String action, final String target, final String value, final String outputPath) {
+        final Long timestamp = System.currentTimeMillis();
+        final String message = String.format("{\"timestamp\": %d, \"pid\": \"%s\", \"action\": \"%s\", \"target\": \"%s\", \"value\": \"%s\"}\n",
+                timestamp, getCurrentPid(), action, target, value);
```

Comment:
We should either escape `value` here to have a valid string also for JSON parsing of the event later on or directly use `Gson` to serialize the `SystemInstrumentationEvent` into a JSON string.